README.md

```diff
@@ -215,6 +215,8 @@ var excel = new ExcelQueryFactory("excelFileName");
 excel.StrictMapping = StrictMappingType.Both;
 ```
 
+If you are using `None` or `ClassStrict` mapping, you can retain unmapped columns by implementing the `IContainsUnmappedCells` interface. This will put all values from the unmapped columns into a dictionary on your class named `UnmappedCells`.
```

Comment:
We should probably include an example here by showing how to implement the interface, and then "using" one of the unmapped values.